docs/iceberg-rest-service.md

```diff
@@ -81,7 +81,63 @@ You can also specify filter parameters by setting configuration entries in the s
 
 ### Security
 
-Gravitino Iceberg REST server supports OAuth2 and HTTPS, please refer to [Security](security/security.md) for more details.
+#### OAuth2
+
+Please refer to [OAuth2 Configuration](./security/how-to-authenticate#server-configuration) for how to enable OAuth2.
+
+When enabling OAuth2 and leveraging a dynamic configuration provider to retrieve catalog information from the Gravitino server, please use the following configuration parameters to establish OAuth2 authentication for secure communication with the Gravitino server:
```

Comment:
Why do we add another configurations instead of reusing existing config options?

Reply:
```
gravitino.authenticators = oauth
gravitino.authenticator.oauth.serviceAudience = test
gravitino.authenticator.oauth.defaultSignKey = xx
gravitino.authenticator.oauth.tokenPath = oauth2/token
gravitino.authenticator.oauth.serverUri = http://localhost:8177
```

Reply:
> ```
> gravitino.authenticators = oauth
> gravitino.authenticator.oauth.serviceAudience = test
> gravitino.authenticator.oauth.defaultSignKey = xx
> gravitino.authenticator.oauth.tokenPath = oauth2/token
> gravitino.authenticator.oauth.serverUri = http://localhost:8177
> ```

This seems to be the configuration for the server side; the configuration added is for the client side.

Reply:
Most Iceberg REST server configuration items are seperated by `-`, I prefer to keep consistent with current style WDYT?